apps/web/pages/team/[slug].tsx

```diff
@@ -27,7 +28,7 @@ function TeamPage({ team }: TeamPageProps) {
   const { isReady, Theme } = useTheme();
   const showMembers = useToggleQuery("members");
   const { t } = useLocale();
-
+  useExposePlanGlobally("PRO");
```

Comment:
We could also look into adding this data into the JWT from session so it should be available to getSession, but not sure if that would work for this anyways, it's the user embedding plan, not logged in one, heh.

But agreed this would be better to be passed somewhere else if most code calling this button won't be consuming it anyways